src/systems/ec/implementations/react/templates/pages/event-detail/examples/Default.jsx

```diff
@@ -0,0 +1,815 @@
+                    }}
+                  />
+                </div>
+              </div>
+
+              <div className="ecl-row ecl-u-type-m ecl-u-mt-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Where
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">
+                  The EGG, Rue Barra 175, 1070 Brussels, Belgium
+                  <br />
+                  <Link
+                    label="See map"
+                    href="/example"
+                    icon={{
+                      shape: 'general--location',
+                      size: 'fluid',
+                    }}
+                  />
+                </div>
+              </div>
+
+              <div className="ecl-row ecl-u-type-m ecl-u-mt-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Live streaming
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">
+                  08:00 CET | 00d:23h:35m
+                </div>
+              </div>
+
+              <div className="ecl-row ecl-u-type-m ecl-u-mt-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Languages
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">
+                  English. Transcripts in Dutch and French
+                </div>
+              </div>
+
+              <div className="ecl-row ecl-u-type-m ecl-u-mt-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Organizer
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">
+                  Diractorate-General Education, Youth, Sport and Culture
+                </div>
+              </div>
+
+              <div className="ecl-row ecl-u-type-m ecl-u-mt-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Event part of
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">
+                  European Year of Culture Heritage
+                </div>
+              </div>
+
+              <div className="ecl-row ecl-u-type-m ecl-u-mt-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Website
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">
+                  <Link label="see website" href="/example" />
+                </div>
+              </div>
+
+              <div className="ecl-row ecl-u-type-m ecl-u-mt-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Social media
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">
+                  {/* :'( Custom social media follow */}
+                  <SocialMediaFollow
```

Comment:
can you fix `SocialMediaFollow` so it doesn't render `ecl-social-media-follow__description` when there's no description please?

Reply:
actually that's an open question. Theoretically the description is mandatory. So this case should not exist.